src/LibPQ.jl

```diff
@@ -26,37 +33,36 @@ const LOGGER = getlogger(@__MODULE__)
 
 function __init__()
     INTERVAL_REGEX[] = _interval_regex()
-    Memento.register(LOGGER)
+    return Memento.register(LOGGER)
```

Comment:
```suggestion
    Memento.register(LOGGER)
    return nothing
```

I don't think there's any way to indicate known function return types to the formatter? Like AFAIK the return value for `__init__()` is always ignored and maybe should always be `nothing`? Not a big deal though.